deploy/charts/trust-manager/templates/webhook.yaml

```
@@ -36,7 +36,11 @@ metadata:
   name: {{ include "trust-manager.name" . }}-tls
   namespace: {{ .Release.Namespace }}
   labels:
-{{ include "trust-manager.labels" . | indent 4 }}
+  {{- include "trust-manager.labels" . | indent 4 }}
```

```suggestion
    {{- include "trust-manager.labels" . | indent 4 }}
```

Nit, both to be more readable, but also for consistency with the other resources.